style.css

```diff
@@ -0,0 +1,90 @@
+div h1 {
+  color: #ff6b00;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+div h2 {
+  color: #fff5e1;
+  font-weight: 700;
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+div p {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+a:link {
+  color: white;
+}
+
+a:visited {
+  color: blue;
+}
+
+a:hover {
+  color: brown;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  background-color: #3c3a39;
+  align-items: center;
+  align-content: space-between;
+  width: 422px;
+  height: 92px;
+  position: absolute;
+  left: 0;
+  top: 0;
+}
+
+.head {
+  background-image: url(img/Group.png);
+  background-repeat: no-repeat;
+  background-position: 292px 60%;
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
```

Comment:
- [ ] Remember you work should be responsive up to the desktop version (768px) kindly set the width here to 100% as well to make the section responsive, ensure the background image is well positioned after making the change
![Screenshot (1022)_LI](https://user-images.githubusercontent.com/65298048/235294531-416b9647-5783-4797-b497-8fa0c71fc160.jpg)